featurebyte/models/deployment.py

```diff
@@ -58,4 +58,4 @@ class FeastIntegrationSettings(BaseSettings):
     Feast integration settings
     """
 
-    FEATUREBYTE_FEAST_INTEGRATION_ENABLED: bool = False
+    FEATUREBYTE_FEAST_INTEGRATION_ENABLED: bool = True
```

Comment:
All the integration tests including databricks_unity run with this flag enabled, is that correct?

Reply:
Yes, all the tests will assume Feast integration is enabled by default unless we explicitly mock it to false.

Reply:
Got it, wanted to confirm since based on our offline discussion the databricks unity integration tests would run with the flag disabled. Seems like that is no longer required.

Reply:
Ya, it is no longer required as this is the only integration test that failed with Feast integration with unity catalog at the moment.